libopae/src/reconf.c

```diff
@@ -133,6 +135,17 @@ static fpga_result validate_bitstream(fpga_handle handle,
 	}
 }
 
+static fpga_result read_port_error(char *errpath, char *path, uint64_t *error)
```

Comment:
I'm not certain that we need to introduce this function here.

How much does this buy us in terms of re-use? I only see this function called twice. Can we just use `sysfs_read_u64()` there?

Reply:
For reading the port error I've abstracted it out so that PR can report if an error exists after a PR is completed/ before unlocking the fpga